utils.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import annotations
-from typing import NamedTuple
-
+from typing import NamedTuple, Union
+from model.object_layer import Object, ObjectLayer
 
 from constants import *
 
+from xml.dom import minidom
+
+coordinate = dict[str, float]
```

Comment:
```suggestion
Coordinate = dict[str, float]
```
uppercase type names :pensive: